applications/views.py

```diff
@@ -30,7 +30,16 @@ def get_queryset(self):
         if self.action == "retrieve":
             return BootcampApplication.objects.prefetch_state_data()
         else:
-            return BootcampApplication.objects.filter(user=self.request.user).all()
+            return (
+                BootcampApplication.objects
+                .filter(user=self.request.user)
+                .select_related("bootcamp_run__bootcamprunpage")
+                .order_by("-created_on")
+            )
+
+    def get_serializer_context(self):
```

Comment:
Should the result of this function be merged with `super.get_serializer_context()`?